chart/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -89,6 +89,8 @@ lms:
   disabled: false
   # if set - always use this region, if empty - region is mapped from the OSB API request
   region: ""
+  # if false - failing LMS step does not break provisioning
+  mandatory: false
```

Comment:
maybe let's set it as `true` as it is production setting and just override it on our CI pipelines.